### ISSUE TITLE
Add support for assets on Terra Blockchain

### ISSUE THREAD

**Issue by bloxster** [rotki/rotki#4043]
## Abstract

Adding Terra assets in Rotki now requires manual input.

## Motivation

Users could just input their Terra address and visualize all assets in their wallet and in staking, popular DeFi assets such as aUSD on Anchor Protocol etc.

## Specification

Instruction on how to connect to Terra APIs are available at [https://algotrading101.com/learn/terra-luna-python-guide/](https://algotrading101.com/learn/terra-luna-python-guide/)

Not sure if this issue is linked to [Issue #2572](https://github.com/rotki/rotki/issues/2572) since Terra is built on Cosmos SDK.

**Comment by stevendavis** [comment]
@bloxster Not sure if it will help you, but you can get the list of transactions on a terra wallet here: https://www.trackterra.org/
Perhaps it could be imported somehow.

**Comment by bloxster** [comment]
Thank you for the link @stevendavis it is very useful indeed, will definitely help me to investigate my Terra account.
Still I hope you will find the way and resources to add this feature to Rotki :-)

**Comment by LefterisJP** [comment]
I don't think we will be adding this chain as it died 2 months after the issue was opened. I know it would have been cool for the few users who used it until it died, but priorities are pressing us towards many other issues.

We will leave it to someone from the community to handle if they can.
